### PR TITLE
0.10.0: Add ability to perform search within text tracks (#297)

### PR DESCRIPTION
**Pull request by wowkalucky**
* Bump __version__ to 0.9.4

* Make transcripts indexable for LMS search (#294)

* Fix javascript unit tests

* Fixed `fullscreen` button overlapping (#295)

* Fix for issue #292: languages selected differ between transcript and caption buttons (#296)

* Update xblock dependecies due to Ginkgo release

* Remove from captions/transcripts control's menu active language highlighting

* Prepare CHANGELOG.md for a new release

* Bump __version__ to 0.10.0

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -50,12 +50,11 @@ def package_data(pkg, roots):
     dependency_links=[
         # At the moment of writing PyPI hosts outdated version of xblock-utils, hence git
         # Replace dependency links with numbered versions when it's released on PyPI
-        'git+https://github.com/edx/xblock-utils.git@v1.0.2#egg=xblock-utils-1.0.2',
-        'git+https://github.com/edx/xblock-utils.git@v1.0.3#egg=xblock-utils-1.0.3',
+        'git+https://github.com/edx/xblock-utils.git@v1.0.2#egg=xblock-utils-1.0.5',
```
Comment (bryanlandia) [inline setup.py:53]: @wowkalucky This line looks wrong, versions between tag and egg don't match.

**Review by bryanlandia — APPROVED** [review]
Nice work!  The text search in transcripts works well!  I think ElasticSearch is tuned for English-based search so searching for text in transcripts in other languages is not very accurate, but that's not an issue with the XBlock I think.  The button layout looks fixed.   

I just made the one comment.  Please check it as it may impact Ginkgo compatibility, which to be honest, I haven't checked.
I posted a few new GitHub Issues I found, but they are unrelated to this PR I think.

https://github.com/raccoongang/xblock-video/issues/298
https://github.com/raccoongang/xblock-video/issues/299

File: setup.py (hunk 1 above)
Comment (wowkalucky) [reply setup.py:53]: It is. To be fixed.

Comment (bryanlandia) [reply setup.py:53]: Thanks @wowkalucky

**Pull request merged**